mrbelvedereci/plan/templates/plan/detail.html

```diff
@@ -92,6 +89,9 @@ <h3 class="slds-heading slds-heading--large">Description</h3>
       <td data-label="Status">
         <div class="slds-truncate slds-badge {% if build.get_status == 'fail' or build.get_status == 'error' %}slds-theme--error{% elif build.get_status == 'success' %}slds-theme--success{% endif %}" title="{{ build.get_status }}">{{ build.get_status }}</div>
       </td>
+      <td data-label="Repository">
+        <div class="slds-truncate" title="{{ build.repo }}">{{ build.repo }}</div>
```

Comment:
seeing ALL the builds for ALL the repos for a plan seems unhelpful.

i think with this schema, we need to introduce r'^(?P<plan_id>\w+)/(?P<repo_owner>\w+)/(?P<repo_name>\w+)/$'` as a list view to show what plan detail currently does now.

Reply:
I wouldn't say it's unhelpful, it will show the most recent builds of the plan which I'd find useful. But I agree it begs for a way to filter down by repo, and your proposed URL format matches the plan run URL format, I like it. I'll add the repo-specific plan view in addition to this.

Reply:
Addressed in https://github.com/SalesforceFoundation/mrbelvedereci/pull/147/commits/dbb8fd5014da8be496a4ebff2197f631e1eaec00